test/integration/footer-links.test.js

```diff
@@ -132,7 +132,7 @@ describe('www-integration footer links', () => {
         expect(pathname).toMatch(/^\/cookies\/?$/);
 
         // Verify localization of last updated message
-        const lastUpdated = await findText('The Scratch Cookie Policy was last updated June 28, 2023');
+        const lastUpdated = await findText('The Scratch Cookie Policy was last updated');
```

Comment:
When this test was added, it was important to see that the date was present and correctly formatted. The policy has been updated since then, though, and I don't think it makes sense to update this test every time we update the policy.